modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingLocalServiceImpl.java

```diff
@@ -1205,13 +1206,9 @@ public CalendarBooking updateCalendarBooking(
 		}
 
 		for (Map.Entry<Locale, String> entry : descriptionMap.entrySet()) {
-			String sanitizedDescription = SanitizerUtil.sanitize(
-				calendar.getCompanyId(), calendar.getGroupId(), userId,
-				CalendarBooking.class.getName(), calendarBookingId,
-				ContentTypes.TEXT_HTML, Sanitizer.MODE_ALL, entry.getValue(),
-				null);
-
-			descriptionMap.put(entry.getKey(), sanitizedDescription);
+			descriptionMap.put(
+				entry.getKey(),
+				_htmlParser.extractText(HtmlUtil.escape(entry.getValue())));
```

Comment:
Hey @G-Sa could you please try to ensure if the a XSS attack is being avoided?

I'd suggest you reproduce the steps of [this bug](https://liferay.atlassian.net/browse/LPD-15082) in this context with this fix. If a XSS attack if being avoided, this change looks good to me :)

Thank you!

Reply:
Thanks! I tried the steps on the ticket and the XSS attack is being avoided!